config.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+
+class Config(object):
+    UNPLANNED_MAINTENANCE = os.getenv('UNPLANNED_MAINTENANCE', False)
```

Comment:
This works for now, but may be worth exploring a more general VIEW var that if not set at all, uses the default, but if set determines which to use. That would allow us to more easily add other pages in the future.

Reply:
I think I should be able to change it to something like `MAINTENANCE_TEMPLATE` or similar and then have a string for each template. We could even do something such as a dictionary with  template strings and the urls as values. Something like {"UNPLANNED_MAINTENANCE":"src/unplanned-maintenance.html". We could then possibly get the value from it and just render the template from that. That can possibly be done at a future date though